v0/destinations/mailchimp/data/mailchimpMergeFieldConfig.json

```diff
@@ -0,0 +1,41 @@
+[
+    {
+      "destKey": "FNAME",
+      "sourceKeys": "firstName",
+      "sourceFromGenericMap": true,
+      "required": false
+    },
+    {
+      "destKey": "LNAME",
+      "sourceKeys": "lastName",
+      "sourceFromGenericMap": true,
+      "required": false
+    },
+    {
+      "destKey": "ADDRESS.addr1",
+      "sourceKeys": ["traits.addr1", "context.traits.addr1"],
+      "required": false
+    },
+    {
+      "destKey": "ADDRESS.city",
+      "sourceKeys": ["traits.city", "context.traits.city"],
+      "required": false
+    },
+    {
+      "destKey": "ADDRESS.state",
+      "sourceKeys": ["traits.state", "context.traits.state"],
+      "required": false
+    },
+    {
+      "destKey": "ADDRESS.zip",
+      "sourceKeys": ["traits.zip", "context.traits.zip"],
+      "required": false
+    },
```

Comment:
Address can be a object as well so update the sourceKeys with `traits.address.**` and `context.traits.address.**`.

Reply:
@sudip but in this case, mailchimp supports some specific keys within the address object. Then are you suggesting keeping support for  "traits.address.zip" also? Otherwise, we won't be able to control what we are sending out